### PR TITLE
**Improvement:** <documentation>

### PR DESCRIPTION
**Pull request by ghost**
index.html created
index.rst updated
README.rst updated

### CODE REVIEW

**Review by yelizariev — APPROVED** [review]

**Review by em230418 — CHANGES_REQUESTED** [review]
Review is still relevant https://github.com/it-projects-llc/misc-addons/pull/736#pullrequestreview-263451492

**Comment by em230418** [comment]
[tests.log](https://github.com/it-projects-llc/misc-addons/files/3446645/tests.log)

**Comment by yelizariev** [comment]
@em230418 a screenshot may help

File: res_partner_skype/static/src/js/test_res_partner_skype.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    var tour = require("web_tour.tour");
+    var core = require('web.core');
+    var _t = core._t;
+
+
+        var steps = [{
+            trigger: '.o_thread_message strong.o_mail_redirect:contains("Agrolait")',
+            content: _t("Open Partners Form"),
+            position: 'bottom',
```
Comment (yelizariev) [inline res_partner_skype/static/src/js/test_res_partner_skype.js:12]: Does it assume that mail module is installed?

**Comment by yelizariev** [comment]
I approve to merge it now

**Pull request merged**